GitUI/CommandsDialogs/SettingsDialog/Pages/AppearanceSettingsPage.cs

```diff
@@ -14,10 +14,16 @@ namespace GitUI.CommandsDialogs.SettingsDialog.Pages
 {
     public partial class AppearanceSettingsPage : SettingsPageWithHeader
     {
+        private const string customAvatarTemplateURL = "https://git-extensions-documentation.readthedocs.io/en/latest/settings.html#git-extensions-appearance-author-images";
```

Comment:
The docs will need to be updated to provide examples on how to configure the custom avatars.

Reply:
This link should be on the provider and describe what the Default provider supplies too

Reply:
I can do that. Should I create a PR for the docs in parallel or after this PR is merged?

Reply:
Do it parallel or after this PR. 
Of course, it is better if the link in the PR does not have to be changed.

Reply:
✔ done
PR can be found here: https://github.com/gitextensions/GitExtensionsDoc/pull/127